prov/shm/src/smr_cq.c

```diff
@@ -35,16 +35,30 @@
 
 #include "smr.h"
 
+static struct fi_ops_cq smr_cq_ops = {
+	.size = sizeof(struct fi_ops_cq),
+	.read = ofi_cq_read,
+	.readfrom = ofi_cq_readfrom,
+	.readerr = ofi_cq_readerr,
+	.sread = ofi_cq_spin_sread,
+	.sreadfrom = ofi_cq_spin_sreadfrom,
+	.signal = fi_no_cq_signal,
```

Comment:
Hmm... we'll want this to force breaking out of the spinning.  This can come as a following patch, once we figure out the best way to implement it.

Reply:
Yeah, I wasn't sure what the best solution for that is...